File: app/src/main/java/com/gallosalocin/go4lunch/services/dto/RestaurantOpeningHours.java
package com.gallosalocin.go4lunch.services.dto;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class RestaurantOpeningHours implements Serializable {

    @SerializedName("open_now")
    private boolean openNow;

    public RestaurantOpeningHours(boolean openNow) {
        this.openNow = openNow;
    }

    public RestaurantOpeningHours() {
    }

    public boolean isOpenNow() {
        return openNow;
    }

    public void setOpenNow(boolean openNow) {
        this.openNow = openNow;
    }

    @Override
    public String toString() {
        return "RestaurantOpeningHours{" +
                "openNow=" + openNow +
                '}';
    }
}

File: app/src/main/java/com/gallosalocin/go4lunch/services/dto/RestaurantOpeningHours.kt
package com.gallosalocin.go4lunch.services.dto

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class RestaurantOpeningHours(
        @SerializedName("open_now")
        var isOpenNow: Boolean = false,
) : Serializable {

//    constructor(openNow: Boolean) {
//        isOpenNow = openNow
//    }
//
//    constructor() {}

    override fun toString(): String {
        return "RestaurantOpeningHours{" +
                "openNow=" + isOpenNow +
                '}'
    }
}